v63004/instream.csh

```diff
@@ -27,13 +27,14 @@
 # gtm8894	    [vinay] Tests that $zreldate outputs in the form YYYYMMDD 24:60
 # gtm8643	    [jake]  Tests that YDB no longer enforces queue depth limit of 5 for SOCKET devices.
 # gtm8922	    [jake]  Tests the functionality of VIEW keywords that take <region-list> expressions
+# gtm8903	    [jake]  Tests $S[ELECT](<TRUE EXPR>:<expr>,<EXPR>:<GLOBAL REFERENCE>) function for errors
```

Comment:
No need to have $S[ELECT] in test. That is needed in documentation to indicate various choices for the same function. $SELECT is fine here.

The test is actually for $SELECT(1:<boolean-expression>,<EXPR>:<GLOBAL REFERENCE>)